tests/common/libvm.sh

```diff
@@ -676,8 +676,9 @@ vm_ostreeupdate_lift_commit() {
 _commit_and_inject_pkglist() {
   local version=$1; shift
   local src_ref=$1; shift
-  vm_cmd ostree commit --repo=$REMOTE_OSTREE -b vmcheck --fsync=no \
-    --tree=ref=$src_ref --add-metadata-string=version=$version
+  # Small percentage by default here
+  vm_cmd rpm-ostree testutils generate-synthetic-update --percentage=5 --repo=$REMOTE_OSTREE --ref=vmcheck \
```

Comment:
```suggestion
  vm_cmd rpm-ostree testutils generate-synthetic-upgrade --percentage=5 --repo=$REMOTE_OSTREE --ref=vmcheck \
```

Reply:
Yeah I have this all fixed locally now, thanks!